<launch>
	<arg name="namespace" default="front_cam"/>
	<arg name="color" default="red"/>
	<arg name="debug" default="true"/>

	<group ns="$(arg namespace)">
		<node
			pkg="base_vision"
			type="board"
			name="board"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="image_rect_color"/>
		<param name="object_color" value="$(arg color)"/>
		<param name="debug" value="$(arg debug)"/>
		<param name="published_topic" value="center"/>

		</node>
	</group>
</launch>
